### ISSUE TITLE
vitejs complains about the browser package

### ISSUE THREAD

**Issue by bomgar** [MasterKale/SimpleWebAuthn#236]
```
@simplewebauthn/browser doesn't appear to be written in CJS, but also doesn't appear to be a valid ES module (i.e. it doesn't have "type": "module" or an .mjs extension for the entry point). Please contact the package author to fix.
```

I don't really know if that is an actual problem.

**Comment by MasterKale** [comment]
The fix for this issue (#237) will go out as v5.4.1 sometime this weekend.

**Comment by MasterKale** [comment]
**@simplewebauthn/browser@5.4.1** has been published that should resolve this issue
